src/transaction/transaction.rs

```diff
@@ -804,10 +889,17 @@ impl TransactionOptions {
             }
         }
     }
+
+    pub fn no_heart_beat(mut self) -> TransactionOptions {
```

Comment:
What about this:

```suggestion
    pub fn stop_heart_beat(mut self) -> TransactionOptions {
```

Reply:
TBO, I think it may be useless. The one who wants to control the heartbeat process will not enable the automatic heartbeat.

Reply:
This function is used to disable auto heartbeat when we start a txn.